drafts/docker-open-liberty-raspberry-pi.adoc

```diff
@@ -1,60 +1,73 @@
+Yasmin Aumeeruddy <https://github.com/yasmin-aumeeruddy>
+
+It is well known that Raspberry Pi's are useful pieces of technology with lots of potential so I set off on a mission to see how easy it is to get a Raspberry Pi-compatible Docker image of Open Liberty. Fortunately Open Liberty is small and lightweight so it was never a problem to fit it on the original Raspberry Pi back in 2012. Although,I have used Raspberry Pi 3's for this demonstration.
 
-This means you can’t take our Open Liberty Docker image from Docker Hub, which is designed for an x86 architecture, and run it on the Pi without any configuration to the Docker images. We need to edit the Dockerfiles that define how to build the x86 Liberty Docker image and modify them to build 
-an image for the ARM Raspberry Pi.
+The complication when moving to a Raspberry Pi is that it has an ARM CPU at its heart, meaning that machine code compiled for x86 or x86-64 CPUs (the standard CPUs you get in laptops/desktops today) won’t execute. Liberty is written in Java so it doesn’t have to worry about what the CPU architecture is: as long as there is a Java runtime it’s able to run (as the Java runtime itself interprets the Java code and executes it on the hardware). We do have official Open Liberty images on Docker Hub, but they are layered on top of the official IBM image, which is built on top of Ubuntu and doesn’t have a JVM that will run on ARM.
 
-=== Open Liberty Images
-To start, take a copy of the link:https://github.com/OpenLiberty/ci.docker/tree/master/official/latest/kernel/java11/openj9[kernel Dockerfile for the x86 Liberty Docker image]. This can be used as the basis for custom built images that contain only the features required for a specific application. 
+This means you can’t take our Open Liberty Docker image from Docker Hub, which is designed for an x86 architecture, and run it on the Pi without any configuration to the Docker images. We need to edit the Dockerfiles that define how to build the x86 Liberty Docker image and modify them to build an image for the ARM Raspberry Pi.
 
-Edit the `FROM` target in the Dockerfile to base the Open Liberty image on an link:https://hub.docker.com/r/arm32v7/adoptopenjdk/[official OpenJDK] image that is compatible with ARM.
+=== Open Liberty images
+To start, take a copy of the kernel Dockerfile for the link:https://github.com/OpenLiberty/ci.docker/tree/master/official/latest/kernel/java12/openj9[x86 Liberty Docker image]. This can be used as the basis for custom built images that contain only the features required for a specific application. 
+Edit the `FROM` target in the Dockerfile to base the Open Liberty image on an link:https://hub.docker.com/r/arm32v7/adoptopenjdk/[OpenJDK] image that is compatible with ARM. For example: 
```

Comment:
"...[OpenJDK image]..." (very minor - but the link is pointing to the image so makes sense to include "image" in the link).